recipes/ucsc-addcols/meta.yaml

```diff
@@ -43,3 +43,7 @@ about:
   home: "http://hgdownload.cse.ucsc.edu/admin/exe/"
   license: "varies; see http://genome.ucsc.edu/license"
   summary: "Sum columns in a text file."
+
+extra:
+  skip-lints:
+    - missing_hash
```

Comment:
Because that requires downloading and checking the hash of 250+ OSX binaries. I did that with a script locally, but I'm not sure you really want to do that in `create_ucsc_packages.py`. That's likely doable, if it's desired.

Reply:
Shall we try compiling on OSX or just putting the checksum in (currently the binaries are just downloaded)? I presume the former, since then at least we're nicely consistent across platforms.

Reply:
We should, but I'm not sure why we do this, @daler had most likely some problems with it.

Reply:
So maybe not worth the effort at this time.

Reply:
OK, I'll hold off on changing that part and focus on ensuring the Linux builds are compiling/passing tests.